common/crypto/openssl/cmac.c

```diff
@@ -44,3 +49,58 @@ oe_result_t oe_aes_cmac_sign(
 
     return result;
 }
+#else
+oe_result_t oe_aes_cmac_sign(
+    const uint8_t* key,
+    size_t key_size,
+    const uint8_t* message,
+    size_t message_length,
+    oe_aes_cmac_t* aes_cmac)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    size_t key_size_bits = key_size * 8;
+    size_t final_size = sizeof(oe_aes_cmac_t);
+    EVP_MAC_CTX* ctx = NULL;
+    EVP_MAC* mac = NULL;
+    OSSL_PARAM params[3];
+
+    if (aes_cmac == NULL)
```

Comment:
We should check `key` and `message` as well (also for the 1.1.1 implementation).